fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/api/event/registry/FabricRegistryBuilder.java

```diff
@@ -34,7 +35,8 @@
  *
  * <pre>
  * {@code
- *  SimpleRegistry<String> registry = FabricRegistryBuilder.createSimple(String.class, new Identifier("registry_sync", "fabric_registry"))
+ *  RegistryKey<Registry<String>> registryKey = RegistryKey.ofRegistry(new Identifier("registry_sync", "fabric_registry"));
```

Comment:
Could probably use a description (maybe "Here is an example of how to create a registry of `String` objects"), and another id such as `new Identifier("mod_id", "registry_id")`.

Reply:
Hopefully thats better now?